store/sqlstore/upgrade.go

```diff
@@ -511,6 +511,13 @@ func UpgradeDatabaseToVersion55(sqlStore SqlStore) {
 	// TODO: Uncomment following condition when version 5.5.0 is released
 	// if shouldPerformUpgrade(sqlStore, VERSION_5_4_0, VERSION_5_5_0) {
 	sqlStore.CreateColumnIfNotExists("PluginKeyValueStore", "ExpireAt", "bigint(20)", "bigint", "0")
+	if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
+		sqlStore.RemoveIndexIfExists("idx_users_email_lower", "lower(Email)")
+		sqlStore.RemoveIndexIfExists("idx_users_username_lower", "lower(Username)")
+		sqlStore.RemoveIndexIfExists("idx_users_nickname_lower", "lower(Nickname)")
+		sqlStore.RemoveIndexIfExists("idx_users_firstname_lower", "lower(FirstName)")
+		sqlStore.RemoveIndexIfExists("idx_users_lastname_lower", "loewr(LastName)")
```

Comment:
Typo:
```suggestion
		sqlStore.RemoveIndexIfExists("idx_users_lastname_lower", "lower(LastName)")
```

Given that the build passes, I guess this means that the migration path isn't tested for PostgreSQL? Something else to investigate.

Reply:
Thanks, applied.
Note I feel like the column name argument probably isn't used, which is why the code worked. To drop an index, the only thing you need is the name of the index.